packages/webpack/src/buildtime/assemble.js

```diff
@@ -41,4 +51,13 @@ const assembleRuntime = (KEY, runtimeModules) => {
   return assembly
 }
 
+/**
+ * @typedef RuntimeModule
+ * @property {string} [file]
+ * @property {any} [data]
```

Comment:
```suggestion
 * @property {unknown} [data]
```

Unless you are writing to `data` or reading its properties, use `unknown`